packages/expo-updates/android/src/main/java/expo/modules/updates/launcher/DatabaseLauncher.kt

```diff
@@ -195,7 +215,7 @@ class DatabaseLauncher(
 
           override fun onSuccess(assetEntity: AssetEntity, isNew: Boolean) {
             database.assetDao().updateAsset(assetEntity)
-            val assetFileLocal = File(updatesDirectory, assetEntity.relativePath)
+            val assetFileLocal = File(updatesDirectory, assetEntity.relativePath!!)
```

Comment:
should this `!!` also be applied to the new callsite on L160? (`File(updatesDirectory, asset.relativePath)`)

Reply:
I'll test that

Reply:
Looks like that works